lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -151,23 +151,43 @@ namespace Js
         }
     }
 
-    uint32 ArrayBuffer::GetByteLengthFromVar(ScriptContext* scriptContext, Var length)
+    uint32 ArrayBuffer::GetToIndex(Var value, int32 errorCode, ScriptContext *scriptContext, uint32 MaxAllowedLength, bool checkSameValueZero)
```

Comment:
Yes they are covered. Look at few at the new tests. Yes test262 got improved.

---
In reply to: [80845333](https://github.com/Microsoft/ChakraCore/pull/1629#discussion_r80845333) [](ancestors = 80845333)